python/mxnet/gluon/model_zoo/vision/mobilenet.py

```diff
@@ -17,27 +17,76 @@
 
 # coding: utf-8
 # pylint: disable= arguments-differ
-"""MobileNet, implemented in Gluon."""
-__all__ = ['MobileNet', 'mobilenet1_0', 'mobilenet0_75', 'mobilenet0_5', 'mobilenet0_25',
-           'get_mobilenet']
+"""MobileNet and MobileNetV2, implemented in Gluon."""
+__all__ = ['MobileNet', 'MobileNetV2', 'mobilenet1_0', 'mobilenet0_75',
+           'mobilenet0_5', 'mobilenet0_25', 'get_mobilenet']
+
+__modify__ = 'dwSun'
+__modified_date__ = '18/01/31'
 
 import os
 
+from ... import nn
 from ....context import cpu
 from ...block import HybridBlock
-from ... import nn
+
 
 # Helpers
+# pylint: disable= too-many-arguments
```

Comment:
This is mobilenet v1 code, add this to pass the style check. Code not changed.

Reply:
please remove and use our pylintrc as standard.

Reply:
ping